aspnetcore/migration/22-to-30.md

```diff
@@ -473,6 +471,8 @@ To use Json.NET in an ASP.NET Core 3.0 project:
                 new CamelCasePropertyNamesContractResolver());
   ```
 
+  **Note:** If you call `AddNewtonsoftJson` without installing [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson), the method fails without throwing an exception. The method also fails silently if you install the wrong Newtonsoft package. Install [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson), not [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/).
```

Comment:
> wrong Newtonsoft package

What is the `wrong Newtonsoft package`? There is only one `Newtonsoft` package (https://www.nuget.org/packages/Newtonsoft.Json/). The reader might get confused with this and think they have the wrong version or something. I wouldn't refer to the aspnet adapter package that lets you get back to using `Newtonsoft` instead of `System.Text.Json` as a "newtonsoft" package.